src/remediations/remediations.queries.js

```diff
@@ -72,7 +76,61 @@ function resolvedCountSubquery () {
     );
 }
 
-exports.list = function (
+function findRunStatus(run) {
+    if (run.count_failure > 0 || run.count_timeout > 0) {
+        return FAILURE;
+    } else if (run.count_running > 0 && run.count_timeout === 0 && run.count_failure === 0) {
+        return RUNNING;
+    } else if (run.count_success > 0 && run.count_timeout === 0) {
+        return SUCCESS;
+    }
+}
+
+async function getremediationStatusCounts() {
+    const dispatcherRuns = await dispatcher_impl.fetchPlaybookRuns({}, {
```

Comment:
I'm pretty sure you don't want to hard-code a bypass to the dispatcher service mocks like this - you'll never be able to test this code path without explicitly overriding the dispatcher connector

const dispatcherRuns = await dispatcher.fetchPlaybookRuns({}, {